MediaBrowser.Model/Configuration/ServerConfiguration.cs

```diff
@@ -254,7 +254,7 @@ public ServerConfiguration()
             AutoRunWebApp = true;
             EnableRemoteAccess = true;
 
-            EnableUPnP = true;
+            EnableUPnP = false;
```

Comment:
![fuck-yeah](https://user-images.githubusercontent.com/1011526/79003924-53b80d00-7b5c-11ea-9cb2-75c4848d893f.gif)

Reply:
Come to think of it... it's disabling UPnP for port forwarding, not all UPnP (DLNA included), right?

Reply:
Just double checked and yes, this only applies to port forwarding. UPnP has it's own configuration: https://github.com/jellyfin/jellyfin/blob/master/Emby.Dlna/Configuration/DlnaOptions.cs

We should really rename it to reflect that, (i.e. `EnablePortForwarding`) but that would be a breaking change for the configuration. Probably better to make a separate PR to rename it and also include a migration for the config

Reply:
I updated the XML documentation for this config property.

I also tried renaming it to `EnablePortForwarding` and writing a migration so the old value of `EnableUPnP` could be maintained. But that doesn't work because the config XML file gets saved and the old value removed before the migrations get a chance to run. I'm thinking it's probably not worth the effort to fix that.